pages/news/2020-10.md

```diff
@@ -0,0 +1,142 @@
+# Declarative-friendly APIs
+
+Welcome to the sixth edition of the **AIP newsletter**, which is designed to
+keep you up to date about the AIP program, and particular proposals making
+their way through the system.
+
+We have an exciting slate of AIPs entering review this cycle, all around the
+theme of "declarative-friendly APIs". The linchpin here is AIP-128, which
+introduces and explains the topic.
+
+The idea behind declarative-friendly APIs addresses an important need: the
+ability to integrate with popular, declarative tools, such as Kubernetes or
+Terraform. These tools and others are used by numerous developers, and the
+follow a straightforward paradigm: the user provides the tool with the desired
+state, and the tool brings that state into fruition.
+
+Integrations with these tools is critical for cloud providers, but with the
+number of services _and_ the number of tools continuously increasing, support
+is difficult without a reliance on automation. Automation, in turn, requires
+strong consistency.
+
+While ordinarily AIPs stay in review for about a month, we intend for this
+review cycle to be longer, to allow time for APIs to be released that follow
+this guidance. Assuming the feedback is sufficiently positive, we will probably
+be looking at formal approval around the end of the year.
+
+## AIPs under review
+
+### AIP-128: Declarative-friendly interfaces
+
+> Many services need to interact with common DevOps tools, particularly those
+> that create and manage network-addressible resources (such as virtual
+> machines, load balancers, database instances, and so on). These tools revolve
+> around the principle of "configuration as code": the user specifies the
+> complete intended landscape, and tooling is responsible for making whatever
+> changes are necessary to achieve the user's specification.
+>
+> These tools are **declarative**: rather than specifying specific actions to
+> take, they specify the desired _outcome_, with the actions being derived
+> based on the differences between the current landscape and the intended one.
+
+[Declarative-friendly interfaces][aip-128] describes patterns for making
+interfaces that are friendly to automated tooling. The basic idea is that by
+being more strict and consistent, it becomes possible to write tools that
+"translate" the interface to a declarative form without having to know anything
+about the specific API.
+
+This reduces feature latency for users. Essentially, the adapter or plugin can
+be automatically generated, and so an automated pipeline becomes possible: as
```

Comment:
"automatically generated or dynamically executed"? (Not sure whether that would be a good thing or not...)

Reply:
Leaving this one as-is.